packages/menhir/menhir.20200121/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "francois.pottier@inria.fr"
+authors: [
+  "François Pottier <francois.pottier@inria.fr>"
+  "Yann Régis-Gianas <yrg@pps.univ-paris-diderot.fr>"
+]
+homepage: "http://gitlab.inria.fr/fpottier/menhir"
+dev-repo: "git+https://gitlab.inria.fr/fpottier/menhir.git"
+bug-reports: "menhir@inria.fr"
+build: [
+  ["dune" "build" "@install" "-j" jobs]
```

Comment:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```
`-p` or `--profile=release` should aways be used for releases.
However mid-way of writing this, I realized you used `-w @1..66` with any profile. Would it be possible to change this?
Apparently previously those were disabled when shipped (https://gitlab.inria.fr/fpottier/menhir/commit/21741c6b14f5ebb0ac5f19852a92d0d20ee7c513#b1842d9251c4973486ce53d049c606e0120f6421_2_0) but this not the case anymore if I understand correctly.

Reply:
Hi Kate, thanks for your comments.

Regarding `-p <name>`, as you noted, it doesn't work, because I install several packages.

Regarding the profile, indeed, as you noted, I do not rely on dune's notion of profile; I just enable a fixed set of warnings (which are non-fatal, I think). Is that a problem?

Reply:
> which are non-fatal, I think

they are fatal, `-w @<spec>` turns them into errors

Reply:
one way of fixing that would be to change `_` by `dev` on this line: https://gitlab.inria.fr/fpottier/menhir/blob/8f0d2b5f2f06762d0f592f753019f92462e7b9c8/src/dune#L4 and to add `--profile=release` to the command in opam-repository

Reply:
I just realized that the flags in `src/dune` were entirely ignored because they were overridden in `src/stage1/dune`, `src/stage2/dune` and `src/stage3/dune`. I need to change the stanzas `(-open MenhirSdk)` to `(flags :standard -open MenhirSdk)`.

Reply:
Done.